test/components/McaLogo.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { render } from "@testing-library/react";
+import React from "react";
+import { McaLogo } from "../../src/components/McaLogo";
+
+describe("McaLogo component", () => {
+  describe("Logo", () => {
```

Comment:
Can we remove this `Logo` describe block here?  For the other components we just have the top level `describe` block for the component and then the one `it` statement